testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/CLIEmbedServerTestCase.java

```diff
@@ -725,8 +725,20 @@ private String readLogOut() {
         return null;
     }
 
-    private void checkLogging(String line) throws IOException {
+    private void checkLogging(String line) throws IOException, InterruptedException {
+        checkLogging(line, 0);
+    }
+
+    private void checkLogging(String line, int timeout) throws IOException, InterruptedException {
         String logOutput = readLogOut();
+        long done = System.currentTimeMillis() + timeout;
+        while (timeout > 0 && System.currentTimeMillis() < done) {
+            if (checkLogging(logOutput, line)) {
+                break;
+            } else {
+                Thread.sleep(20);
+            }
```

Comment:
Thanks for adding this, although I would suggest a longer pause, let's say 1 second or half one. Using TimeUnit makes it more readable, but that's also completely optional e.g `TimeUnit.MILLISECONDS.sleep(500)`

The else can be removed since the break will do the work, that's optional too.

None of these notes would prevent merging it as it is, so I leave it up to you to apply the changes.

Reply:
Thanks for review @yersan, I have added suggested changes.